Source/SIMPLib/CoreFilters/ImportAsciDataArray.cpp

```diff
@@ -190,7 +190,7 @@ template <typename T, typename K> int32_t readAsciFile(typename DataArray<T>::Po
       {
         return err;
       }
-      if(err == RBR_READ_ERROR || err == RBR_READ_ERROR)
+      if(err == RBR_READ_ERROR || err == RBR_READ_ERROR) // Identical
```

Comment:
They are the same but I wonder if the code should be checking another error condition?